docs/assemblies/assembly-prometheus-metrics-proxy.adoc

```diff
@@ -0,0 +1,79 @@
+// file included in the following:
+//
+// kroxylicious-operator/index.adoc
+
+[id='assembly-prometheus-metrics-proxy-{context}']
+= Proxy Metrics
+
+[role="_abstract"]
+
+The proxy provides *connections metrics* and *message metrics*.
+
+Connection metrics allow the connections made from the clients to the proxy (the downstream) and the connection made by the proxy to the kafka cluster (the upstream) to be understood.
```

Comment:
Is it worth calling out, not necessarily here, that we expect those metrics to be in sync? I.e. seeing `client_to_proxy=100` should also see `proxy_to_server~=100`

Reply:
Done